src/lib/transition_frontier_controller_tests/stubs.ml

```diff
@@ -123,14 +123,22 @@ struct
         let%bind sender_sk = sender_sk in
         let sender_keypair = Keypair.of_private_key_exn sender_sk in
         let%bind receiver_pk = List.random_element public_keys in
+        let nonce =
+          let ledger = Staged_ledger.ledger staged_ledger in
+          let status, account_location =
+            Ledger.get_or_create_account_exn ledger sender_account.public_key
+              sender_account
+          in
+          assert (status = `Existed) ;
```

Comment:
Are we OK with using polymorphic compare here?

Reply:
Yes. Otherwise, how would you use equality on a polymorphic variant? `=` should be fine for that case.

Reply:
You can derive equality for polymorphic variants if you really want, but I would agree that in this case polymorphic comparison is fine. It would only matter if there was some data nested under the constructor you are comparing against (like, if it was `assert (status = (`Existed x))`).